bob/extension/utils.py

```diff
@@ -305,6 +308,11 @@ def find_executable(name, subpaths=None, prefixes=None):
 
   binpaths += ['bin']
 
+  # Windows
+  if os.name == 'nt':
+    binpaths += [os.path.join('mingw-w64','bin')]
+    name = name + '.exe'
+
```

Comment:
You probably want to remove this too. We are trying to add support for msvc not mingw 64 which are incompatible

Reply:
I don't think incompatibility applies to executables. Many executables are compiled on MingW and can be used by passing input arguments regardless of how they are compiled. Removing this will cause problems in the future, if not now. The "name = name + '.exe'" is also required as in windows executables need the extension in special cases.